### ISSUE TITLE
DOMException: BodyStreamBuffer was aborted

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#2960]
## tl;dr

Hitting the Back button in a particular spot can trigger a stack trace. While the trace is prominent when running in Dev mode, in production it only appears in the DevTools console and doesn't seem to impact the functionality of the Back button, so it's probably a low priority to figure out.

## Details

Repro is with Zui Insiders [1.5.1-9](https://github.com/brimdata/zui-insiders/releases/tag/v1.5.1-9) which is based on Zui commit 05f25cd.

This issue was originally spotted during [PR testing](https://github.com/brimdata/zui/pull/2895#issuecomment-1867085303) but @jameskerr concluded at the time:

> This needs more time to figure out. I'm not sure why this is happening and it's very difficult to debug and test because of the timing required to trigger it. Also in production, this error wouldn't pop up, it would just appear in the dev tools console.

Repro steps as shown in the video below:

1. Load the [conn.zng.gz](https://github.com/brimdata/zed-sample-data/blob/main/zng/conn.zng.gz) sample data
2. Click to get into Inspector mode
3. Select "Count by this field" on a `ts` value
4. Wait a few seconds
5. Click the Back button

https://github.com/brimdata/zui/assets/5934157/1fa89f87-c946-4bc2-bb44-da6decf2d2fc

Given @jameskerr's comments about the difficulty reproducing, here's a couple things I've confirmed during these more recent repros in terms of how I can reliably make it happen:

* It seems to **only** repro if the a `ts` value is the one I right-click for **Count by this field**.  So for instance if I right-click on a `_path` value it does _not_ seem to repro.
* It seems to repro best if I **wait a few seconds** after clicking **Count by this field** before I click the Back button. If I pounce on the Back button too fast it doesn't seem to repro.

Here's the full stack trace that's shown in DevTools:

```
Uncaught (in promise) DOMException: BodyStreamBuffer was aborted
    at AbortSignal.<anonymous> (file:///_next/static/chunks/894-5b8a75273572831d.js:1:133856)
    at AbortSignal.<anonymous> (file:///_next/static/chunks/894-5b8a75273572831d.js:1:133856)
    at Object.abort (file:///_next/static/chunks/894-5b8a75273572831d.js:1:36165)
    at file:///_next/static/chunks/894-5b8a75273572831d.js:1:27326
    at Array.map (<anonymous>)
    at s.abort (file:///_next/static/chunks/894-5b8a75273572831d.js:1:27317)
    at X.createAbortable (file:///_next/static/chunks/894-5b8a75273572831d.js:1:36043)
    at X.query (file:///_next/static/chunks/894-5b8a75273572831d.js:1:36353)
    at async E (file:///_next/static/chunks/894-5b8a75273572831d.js:1:105957)
    at async P (file:///_next/static/chunks/894-5b8a75273572831d.js:1:106560)
(anonymous) @ 894-5b8a75273572831d.js:1
(anonymous) @ 894-5b8a75273572831d.js:1
abort @ 894-5b8a75273572831d.js:1
(anonymous) @ 894-5b8a75273572831d.js:1
abort @ 894-5b8a75273572831d.js:1
createAbortable @ 894-5b8a75273572831d.js:1
query @ 894-5b8a75273572831d.js:1
```
